templates/kubernetes/charmed-k8s.html

```diff
@@ -2,348 +2,418 @@
-        <li class="p-list__item is-ticked">Bare-metal performance</li>
-        <li class="p-list__item is-ticked">Guaranteed full-stack high availability</li>
-        <li class="p-list__item is-ticked">DevOps-driven bare metal management</li>
-        <li class="p-list__item is-ticked">Optimised hardware utilisation and elastic machine allocation</li>
-        <li class="p-list__item is-ticked">Enhanced platform awareness for GPU auto-detection</li>
-        <li class="p-list__item is-ticked">Fault-tolerant software-defined storage</li>
-      </ul>
+    <div class="u-fixed-width">
+        <hr class="p-separator" />
     </div>
-    <div class="col-7 u-hide--medium u-hide--small u-align--center u-vertically-center">
-      {{
-        image (
-        url="https://assets.ubuntu.com/v1/e279ccc1-kubernetes-on-bare-metal.svg",
-        alt="",
-        width="607",
-        height="213",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
+
+    <div class="row">
+        <div class="col-5">
+            <h2>Kubernetes on bare metal</h2>
+            <p class="p-heading--4">Optimal performance, management and automation of physical and container assets</p>
+            <p>
+                Charmed Kubernetes seamlessly runs on bare metal in conjunction with MAAS, Canonical's bare metal cloud orchestrator. MAAS offers a cloud-like experience for K8s deployments on the most popular hardware infrastructure. Charmed Kubernetes on MAAS provides:
+            </p>
+            <ul class="p-list">
+                <li class="p-list__item is-ticked">Bare-metal performance</li>
+                <li class="p-list__item is-ticked">Guaranteed full-stack high availability</li>
+                <li class="p-list__item is-ticked">DevOps-driven bare metal management</li>
+                <li class="p-list__item is-ticked">Optimised hardware utilisation and elastic machine allocation</li>
+                <li class="p-list__item is-ticked">Enhanced platform awareness for GPU auto-detection</li>
+                <li class="p-list__item is-ticked">Fault-tolerant software-defined storage</li>
+            </ul>
+        </div>
+        <div class="col-7 u-hide--medium u-hide--small u-align--center u-vertically-center">
+            {{
+            image (
+            url="https://assets.ubuntu.com/v1/e279ccc1-kubernetes-on-bare-metal.svg",
+            alt="",
+            width="607",
+            height="213",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+            }}
+        </div>
     </div>
-  </div>
-  <div class="row">
-    <div class="col-12">
-      <a href="https://technology.amis.nl/platform/kubernetes/charmed-kubernetes-on-kvm-using-maas-and-juju/">Learn more about MAAS&nbsp;&rsaquo;</a>
+    <div class="row">
+        <div class="col-12">
+            <a href="https://technology.amis.nl/platform/kubernetes/charmed-kubernetes-on-kvm-using-maas-and-juju/">Learn more about MAAS&nbsp;&rsaquo;</a>
```

Comment:
This is the only line of code that actually needs reviewing, everything else was just the jinja linter

Reply:
@mtruj013 looks like it's not this line that you changed :)